packages/factory/src/helpers.ts

```diff
@@ -118,7 +118,9 @@ export const getContext = ({
         },
     };
 
-    const passedEnv: Env = (process.env.BUILD_PLUGINS_ENV as Env) || 'development';
+    // Use "production" if there is no env passed.
+    const passedEnv: Env = (process.env.BUILD_PLUGINS_ENV as Env) || 'production';
+    // Fallback to "development" if the passed env is wrong.
```

Comment:
Just curious, what's the difference between development and production here? Log verbosity? Something more?

Reply:
Right now, the main diff is logging to Datadog in the analytics plugin, only done in `production`.

About this change specifically: 
Before, it was always falling back to `development` when not defined.
And I forgot to explicitly define it when bundling and publishing it.
Resulting with a `development` env in actual production, so got me thinking that maybe the better default is `production`.
Jury's still out though.